config/settings/base.py

```diff
@@ -206,6 +206,10 @@
 ]
 # http://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-broker_url
 CELERY_BROKER_URL = env("CELERY_BROKER_URL", default="django://")
+# https://docs.celeryproject.org/en/stable/userguide/optimizing.html#broker-connection-pools
+CELERY_BROKER_POOL_LIMIT = env(
+    "CELERY_BROKER_POOL_LIMIT", default=env("CELERYD_CONCURRENCY", default=500)
+)
```

Comment:
How is this used in the application?

Reply:
It's used on the client side, as the other `CELERY_BROKER` configuration